lib/netbox_client_ruby/communication.rb

```diff
@@ -32,43 +32,60 @@ def hash_to_object(hash)
 
     private
 
+    def sanitize_variable_name(raw_name)
+      raw_name.gsub(/[^a-zA-Z0-9_]/, '_')
```

Comment:
I think it's better the way it is, because otherwise you will have to know exactly what `\w` means to understand it, while here you see it at one glance what the regex is supposed to do.

Also, I did not expect `\w` to match the `_`.